tests/pkg/k8s/jobs.go

```diff
@@ -127,3 +131,42 @@ func WaitForJob(t *testing.T, ctx context.Context, k8sClient *kubernetes.Clients
 		time.Sleep(time.Second * 5)
 	}
 }
+
+func PrintJobLogs(t *testing.T, ctx context.Context, k8sClient *kubernetes.Clientset, jobName, namespace, containerName string) {
```

Comment:
can we somehow get the pod's logs only if it fails and have error? Or the logs from these jobs(system/tenant fetcher, cert-rotation-job, etc..) are not too verbose and it won't be a problem?